unitylibs/core/workflow/workflow-acrobat/target-config.json

```diff
@@ -1,12 +1,17 @@
 {
   "acom-widget": {
```

Comment:
Think we can go ahead and update this to `acom-widget.fillsign`.

Reply:
Not updating this as of now I see Blaine's comment and we might need to do more changes in config as the block name is changed